### PR TITLE
Change back to the regular logo

### LINKED ISSUE #913: Replace logo image if needed it
Aiven will support and celebrate pride month for the next 11 months as you can fin here: https://aiven.io/blog/celebrating-our-colorful-cast 

The images were replaced in this PR: https://github.com/aiven/devportal/pull/909 

This issue should be revisited in a year (June 2023) to check if the images should be replaced back or not.

### PR DESCRIPTION
**Pull request by FloorD**
# What changed, and why it matters

Aiven Developer sported the Pride version of our logo for the June month. This PR takes us back to the before times ⌚ 

Closes #913

### CODE REVIEW

**Review by laysauchoa — DISMISSED** [review]
Thanks for the PR, Floor! 

Does this address this Issue: https://github.com/aiven/devportal/issues/913? If yes, could you please link to this PR? It is helpful because it will automatically close this PR, once it is accepted by a reviewer.

Also, is the image only for the June month? I understood that it was for the whole year in the linked article. The Aiven console is with the pride image while the aiven website not, so I am not sure which one is correct.

File: _static/css/aiven.css [hunk 1]
```diff
@@ -186,7 +186,7 @@ main > .content p:not(.card-text) a:not(.btn) {
     width: 80%;
     margin: 24px auto 10px;
     padding-top: 5px;
-    background-image: url('../../_static/images/logoCrabbyDark-pride.svg') !important;
```
Comment (anelook) [inline _static/css/aiven.css:189]: Should we also remove the file itself?

**Comment by lornajane** [comment]
Can we also update the preview image? Or open another ticket to do so.

**Comment by lornajane** [comment]
Here are the changes for when we added it, for reference https://github.com/aiven/devportal/pull/909/files

**Pull request merged**